### DISCUSSION TITLE
Large mdmp file in bin folder

### DISCUSSION THREAD

**Discussion by adriancoca** [orientechnologies/orientdb#9714] [Q&A] [answered]
## Problem

I found that i have a big file of 48GB with extension mdmp. I think it is a dump file.

It is safe to delete this 4 files ?

![image](https://user-images.githubusercontent.com/7643185/139822704-7f8593f6-866f-49bb-8806-850f775eea6b.png)

**Comment by andrii0lomakin** [comment, marked as answer]
Yes, absolutely we do not use such files. Please use the forum to ask questions, next time.
